### ISSUE TITLE
Test transaction for token swap worked, larger transaction failed, over 40K XLC not reporting

### ISSUE THREAD

**Issue by roryboreyalice** [leviarcoin/leviarcoin-token#1]
I setup my account on the leviar platform website.  From there, I used the wallet address and payment ID to send 5 XLC to the token swap. The transaction reported instantly on the block explorer, and made it to my leviar account about an hour later.  Since the test transaction worked, I sent a handful of ~5000 XLC transactions.  When I sent those transactions, I didn't type anything new, I hit the up arrow, and modified the previous transaction amount, and reissued the same command with the same wallet/payment ID.  All of the transactions posted to the block explorer (email me at my [username]@gmail.com for wallet/transactions).  Not a single transaction has posted to the token swap dashboard.  That's over 40K XLC.  It has been about 20 hours now, there have been over 40 blocks mined.  What is causing the delay?  Where is my XLC?

_(edited for clarity)_

**Comment by BKdilse** [comment]
Yeah, I've seen similar behavior.  Sent 10 XLC, confirmed, then sent 20 XLC, confirm, then 1000 (which never showed.
Since then, I've sent a smaller amount of 60 XLC, which has shown up, but does not seem to be eligible for the swap.

What is going on here, where is our XLC gone?  and why is the balance eligible, different from what's shown in the transactions?

**Comment by roryboreyalice** [comment]
It's been a couple days with no response, and no activity on my token swap wallet.  I can only guess I lost everything.  Well handled gang.

**Comment by roryboreyalice** [comment]
Still nothing from the devs.  I got my 5 XLP now.  Yay, 40K+ Totally gone.

**Comment by BKdilse** [comment]
I'll also received mine, looks buggy.  Dev can you respond?
